openapi/components/paths/users.yaml

```diff
@@ -174,6 +174,29 @@ paths:
           $ref: ../responses/MissingCredentialsError.yaml
       security:
         - authCookie: []
+  "/users/{userId}/groups/represented":
+    parameters:
+      - $ref: ../parameters.yaml#/userId
+    get:
+      summary: Get user's current represented group
+      description: Returns the current group that the user is currently representing
+      operationId: getUserRepresentedGroup
+      tags:
+        - users
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                items:
```

Comment:
Items is not a valid object property.
https://swagger.io/docs/specification/data-models/data-types/#object